docs/worker.md

```diff
@@ -44,6 +44,9 @@ To start any Symfony application in a worker, install the FrankenPHP package of
 composer require runtime/frankenphp-symfony
 ```
 
+> [!TIP]
+> Starting with Symfony 7.4, the `runtime/frankenphp-symfony` package isn't required anymore, as Symfony Runtime natively supports FrankenPHP worker mode.
```

Comment:
```suggestion
> [!TIP]
> The following section is only necessary if you are using an older Symfony version below 7.4.
```
So that users with new versions can completely skip that part and don't need to learn about the runtime (or install it to their composer dependencies).